### DISCUSSION TITLE
Compress in Arm 32

### DISCUSSION THREAD

**Discussion by VenkatNeeraj** [upx/upx#701] [General]
when i am compressing a Arm 32 Elf its giving output in Arm 64
I tried using upx 32 exe in windows and also using -8086 but the results are same 
how can i compress in Arm 32 ?

**Comment by jreiser** [comment]
Please supply a repeatable test case with specific details, such as the exact literal commands used, and the version of UPX (`upx --version`).  Please upload ("Attach files by ...") the `Arm 32 Elf` that is being compressed, and the `output in Arm 64`.  Compress the uploads using zip, gzip, or .tgz to satisfy GitHub; or supply URLs to the files.
